fix: read the error status of `App.Error` in the `prerender` and `query.live` remote functions (#16529)
sveltejs/kit: .changeset/remote-error-status.md

---
'@sveltejs/kit': patch
---

> chore: read the error status from `App.Error` in the remaining remote-response readers

Reviewer: ```suggestion
fix: read the error status of `App.Error` in the `prerender` and `query.live` remote functions
```
